documentation/cloud-manager-guide/secure-data-connectors/manage-data-connectors.md

```diff
@@ -0,0 +1,66 @@
+---
+keywords:
+title: Manage Data Connectors
+description: Manage data connectors
+---
+# Manage Data Connectors
+
+Secure data connectors can be managed from the *Secure Data Connectors* screen. From here, you can edit, enable or disable, and delete connectors. This guide outlines the steps to manage data connectors.
+
+## Getting started
+
+All of the operations to manage secure data connectors are available in the **Options** (**...**) dropdown for each connector. To begin managing a connector, select the **Options** (**...**) menu of specific connector you wish to modify.
+
+**image**
+
+## Edit a data connector
```

Comment:
Let's clarify here, "Edit data connector name and description" since I think that's all this section is about.